doc/cephfs/index.rst

```diff
@@ -35,173 +53,3 @@ other storage interfaces to form a modern unified storage system: RBD (Ceph
 Block Devices) and RGW (Ceph Object Storage Gateway).
 
 
-Getting Started with CephFS
-^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-For most deployments of Ceph, setting up a CephFS file system is as simple as:
-
-.. code:: bash
-
-    ceph fs volume create <fs name>
-
-The Ceph `Orchestrator`_  will automatically create and configure MDS for
-your file system if the back-end deployment technology supports it (see
-`Orchestrator deployment table`_). Otherwise, please `deploy MDS manually
-as needed`_.
-
-Finally, to mount CephFS on your client nodes, see `Mount CephFS:
-Prerequisites`_ page. Additionally, a command-line shell utility is available
-for interactive access or scripting via the `cephfs-shell`_.
-
-.. _Orchestrator: ../mgr/orchestrator
-.. _deploy MDS manually as needed: add-remove-mds
-.. _Orchestrator deployment table: ../mgr/orchestrator/#current-implementation-status
-.. _Mount CephFS\: Prerequisites: mount-prerequisites
-.. _cephfs-shell: cephfs-shell
```

Comment:
I'd rather keep this section in `index.rst`.

Reply:
We'll have to find a way to keep this in index.rst while preserving its location in the sidebar link. Currently, if we keep the rst as you have quoted here, a link to "Getting Started with CephFS" will appear at the end of the list of items under "Administration" in the sidebar menu.

I'll see what we can do about this. Expect more back-and-forth.